.github/workflows/spec.yml

```diff
@@ -0,0 +1,31 @@
+name: spec
+on:
+  push:
+    paths:
+      - '**.bs'
+  pull_request:
+    paths:
+      - '**.bs'
+jobs:
+  specification:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - uses: ruby/setup-ruby@477b21f02be01bcb8030d50f37cfec92bfa615b6
+        with:
+          ruby-version: 2.6
+      - name: Install dependencies
+        run: |
+          pip install bikeshed xml2rfc python-dateutil
+          bikeshed update
+          if head -1 -q *.md | grep '^\-\-\-' >/dev/null 2>&1; then gem install --no-doc kramdown-rfc2629; fi
+          if head -1 -q *.md | grep '^%%%' >/dev/null 2>&1; then go get "$mmark_src" && go build "$mmark_src"; fi
+      - name: Make github pages
+        run: |
+          bikeshed --die-on=warning spec loading.bs
+          make
```

Comment:
It'll probably be more straightforward if we split the Bikeshed build and gh-pages upload from the kramdown-rfc2629 one, into separate files. The kramdown-rfc2629 build can use https://github.com/martinthomson/i-d-template/blob/master/template/.github/workflows/ghpages.yml, and the Bikeshed build should use https://w3c.github.io/spec-prod/#deploy-to-github-pages.

`--die-on=warning` becomes https://w3c.github.io/spec-prod/#build_fail_on.

Reply:
Thanks. I've split this file into ghpages.yml and spec-prod.yml.